src/views/PersonProfile/PersonProfile.js

```diff
@@ -90,16 +104,22 @@ export default function PersonProfile() {
                 <GridItem xs={12} sm={12} md={2}>
                   <CustomInput
                     labelText="Surname"
+                    labelProps={{name: "lastName"}}
                     id="surname"
                     formControlProps={{
                       fullWidth: true
                     }}
                     onChange={(event)=> setPerson({...person, surname: event.target.value})}
                     />
                 </GridItem>
-                <GridItem xs={12} sm={12} md={1}>
+                <GridItem xs={12} sm={12} md={3}>
+                  {/* I changed md to 3 because some of the text disappears at 2 */}
                   <CustomInput
-                    labelText="Birthday"
+                    labelText="Birth date"

```

Comment:
I dont know if we should use "Birth date" instead of "Birthday". Because in the class diagram "Person" entity has "Birthday" field, so if we will use "Birth date" we will introduce a place for interpretations and possible miscommunication

Reply:
Birthday and birth date are two different things.  A birthday is an annual celebration and birth date is the thing we are referring to in this case.

If you want to support multiple languages, you could just leave birthday as the key for the localization mapping and the value for English would be birth date.